modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -14251,6 +14516,78 @@ private void _testGetObjectEntryWithObjectActions(
 		unsafeTriConsumer.accept(actionJSONObject, jsonObject, objectAction);
 	}
 
+	private void _testPatchPutScopeScopeKeyByExternalReferenceCode(
+			Http.Method httpMethod)
+		throws Exception {
```

Comment:
I think we can add an `ObjectDefinition objectDefinition` parameter, similar to what's done in `_testPatchPutCustomObjectEntryWithAttachmentField()`. 

This would make the method more reusable, as it would allow us to add an object entry based on the provided object definition. Since object definitions can have different scopes, this change would make the test more flexible and easier to reuse in the future.